src/Components/ActionCard/ActionCard.js

```diff
@@ -8,11 +8,10 @@ const ActionCard = ({ isEdit = false, isDelete = false, id, onSave = () => { },
         <div className="card">
             <div className="card-header">
                 <div className="float-end">
-                    {isEdit &&
-                        <i className="fa fa-pencil ml-2"
-                            style={{ fontSize: "24px", color: 'orange' }}
-                            onClick={() => {onEdit(id); setSaveFlag(true)}}></i>}
-                    {isDelete && <i className="fa fa-trash ml-2" style={{ fontSize: "24px", color: 'red' }} onClick={() => onDelete(id)}></i>}
+                    {isEdit && <i className="fa fa-pencil ml-2" style={{ fontSize: "24px"}} 
```

Comment:
Try to avoid to write inline styles

Reply:
its still not addressed, I see fontsize 24 used in multiple places even Yasthika using it in her branch, can you make one class for it and use it
naming convention fs-24  --> fs means fontsize, 24 means size, 
pt-24 --> padding top 24px